perl/Git/SVN/Fetcher.pm

```diff
@@ -142,32 +144,27 @@ sub is_path_ignored {
 ############################################################
 
 =item is_empty_commit()
-
 Return 1 if all given $paths are ignored, so that this commit end up in an empty commit
-
 Input:  $path - array of strings (Paths) in a commit
-
 Output: { 1 if true, 0 if false }
-
 =cut
 
 ############################################################
-sub is_empty_commit{
-	my ($self, $paths) = @_;
-	my $path = "";
-	unless (defined($self->{include_regex})){
-		return 0;
-	}
+sub is_empty_commit {
+    my ( $self, $paths ) = @_;
+    my $path;
+	my $ignored;
+    unless ( defined( $self->{include_regex} ) ) {
+        return 0;
+    }
 
-	foreach $path (keys %$paths){
-		unless (defined $path && -d $path ){
-			my $ignored = $self->is_path_ignored($path);
-			if (!$ignored){
-				return 0;
-			}
+    foreach $path ( keys %$paths ) {
+		$ignored = $self->is_path_ignored($path);
+		if ( !$ignored ) {
+			return 0;
 		}
-	}
-	return 1;
+    }
+    return 1;
```

Comment:
There are no performance changes I see in this commit, but a lot of bogus white-space changes... And no sign-off, and the commit message does not really say what this is about. Please address this.

Reply:
Done see below. for Soem reason the whole file was had this issue.

Reply:
Could you use an [interactive rebase](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d) to turn this into a clean patch? In the Git project, no patch series is accepted that introduces a bug that is fixed in the same patch series.